hazelcast/src/main/java/com/hazelcast/jet/impl/JetServiceBackend.java

```diff
@@ -173,15 +173,13 @@ public void configureJetInternalObjects(Config config, HazelcastProperties prope
         config.addMapConfig(internalMapConfig)
                 .addMapConfig(resultsMapConfig)
                 .addMapConfig(metricsMapConfig)
-                .addMapConfig(initializeSqlCatalog(config.getMapConfig(SQL_CATALOG_MAP_NAME)));
+                .addMapConfig(initializeSqlCatalog(new MapConfig()));
```

Comment:
There's no point in passing the map config to the method. I'd rename the method to `createSqlCatalogConfig` and remove its parameter.

Reply:
changed